activestorage/lib/active_storage/attached/model.rb

```diff
@@ -53,7 +60,7 @@ def #{name}=(attachable)
               if attachable.nil?
                 ActiveStorage::Attached::Changes::DeleteOne.new("#{name}", self)
               else
-                ActiveStorage::Attached::Changes::CreateOne.new("#{name}", self, attachable)
+                ActiveStorage::Attached::Changes::CreateOne.new("#{name}", self, attachable, #{key})
```

Comment:
`key` is not in a string interpolation context (avoid `#{}`)

Maybe out of PR context and/or contrary to Rails guidelines: why not using `name.to_s` instead of `"#{name}"`?

Reply:
My apologies, I was not aware that those lines of code was in a heredoc context!